README.md

```diff
@@ -1,58 +1,53 @@
 # Tailscale package for Synology NAS
-![CI](https://github.com/tailscale/tailscale-synology/workflows/CI/badge.svg)
 
-Synology NAS package for Tailscale based on precompiled static binaries.
-
-## Disclaimer
-
-You use everything here at your own risk. Make sure you have other network
-paths to your NAS before installing this, in case something goes wrong.
+Synology NAS package for Tailscale.
 
 ## Issue Tracker
 
 File issues at: https://github.com/tailscale/tailscale/issues
 
-This repo's issue tracker is disabled. (And all historical issues have been moved so the old URLs redirect)
-
 ## Installation
 
-1.  Download precompiled [releases](https://pkgs.tailscale.com) from the page for SPKs for your platform.
+1.  Download precompiled [releases](https://pkgs.tailscale.com) from
+    the page for SPKs for your platform.
 2.  In the Synology DSM web admin UI, open the Package Center.
 3.  Press the *Manual install* button and provide the SPK file.
 4.  Follow the wizard until done.
 5.  At this point `tailscaled` should be up and running.
-6.  SSH into the  machine, and run `sudo tailscale up` so you can authenticate.
-
-## Upgrading
-
-If upgrading to version v1.10.0, you may end up with duplicate installations of Tailscale. This is a [known](https://github.com/tailscale/tailscale/issues/2266#issuecomment-869792505) side effect of some metadata changes that were made in v1.10.0 in preparation of the installation package to be listed in the Synology Package Center. It is recommended to uninstall the old Tailscale package first before upgrading to v1.10.0. Please note that your devices Tailscale IP may change when v1.10.0 is installed.
-
-## Compatibility
+6.  SSH into the machine, and run `sudo tailscale up` so you can
```

Comment:
...I guess we haven't updated this in a while. They don't need to ssh now, they click on the Tailscale icon in the Package Center and then click "open" to be taken to a web page where they can authenticate.

Reply:
I replaced this entire section with a link to https://tailscale.com/kb/1131/synology/ , which is more up to date and also covers installation via the official packaging center.